### DISCUSSION TITLE
Color-tagged text does not work in interaction labels

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15384] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

![20241216222735](https://github.com/user-attachments/assets/25309cfe-b9ab-411b-82d7-298d904cf9c7)

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested a fix for this.
Closing discussion as resolved.
